addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -119,6 +119,10 @@ private _turretPaths = ((fullCrew [_vehicle, "gunner", true]) + (fullCrew [_vehi
 
         // Find the action position
         private _position = compile format ["_target selectionPosition ['%1', 'HitPoints'];", _selection];
+        if ("rotor" in _hitpoint) then {
```

Comment:
is this naming convention enforced by the engine or is GM gonna be special again?

Reply:
oh, haven't even thought of that - sadly I have no idea and no way to check GM
all the testing I've done was on vanilla and RHS, there it seems that 100% rotors do have rotor somewhere in the name.

Reply:
Seems to be enforced by the engine https://community.bistudio.com/wiki/Arma_3:_Hitpoints

Reply:
Nevermind, I think I misread something - running the script from the link shows that just with RHS the hitpoint list grows to 600+, so this will have to be checked for GM (RHS and CUP seem ok here)